resources/windows/msi/service.wxs.erb

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+<!-- include the set of service files -->
+
+  <%- service_files.each do |service| -%>
+    <?include $(sys.SOURCEFILEDIR)include/<%=  service.path %>' ?>
```

Comment:
The `include` element is meant for `wxi` files, not `wxs` files. I haven't personally seen this work, although documentation implies it might. I haven't seen this as a common practice.

In the past I have never used this, instead passed the list of `wxs` files to light either with a `*.wxs` or a listing of each file.